openvm/src/customize_exe.rs

```diff
@@ -454,6 +457,93 @@ pub fn openvm_bus_interaction_to_powdr<F: PrimeField32, P: FieldElement>(
     Ok(SymbolicBusInteraction { id, mult, args })
 }
 
+#[derive(Serialize, Deserialize)]
+struct ApcCandidate<P, T> {
+    block_with_apc: BlockWithApc<P, T>,
+    execution_frequency: usize,
+    cells_saved_per_row: usize,
+    width: usize, // only tag this field in Pgo::Cell, the only place it's needed
+}
+
+impl ApcCandidate<BabyBearField, OpenVmField<BabyBearField>> {
+    pub fn try_create(
```

Comment:
Moved into this function, and added the candidates dir path

Reply:
minor: maybe `save_to_disk` could be called separately from creating the candidate (instead of extra arg here)?